share/completions/loginctl.fish

```diff
@@ -30,7 +30,7 @@ complete -c loginctl -f -n "not $seen $commands" -l version -d "Show package ver
 
 
 function __fish_loginctl_list_sessions
-    loginctl list-sessions --no-legend --no-pager --output=short | string replace -r '(\d+) \d+ (\S+) \S+ (\S+) .*' '$1\t$2 at $3'
+    loginctl list-sessions --no-legend --no-pager --output=short | string replace -r '(\d+) \d+ (\S+) (\S*|\s+) (\S+) .*' '$1\t$2 at $4'
```

Comment:
Neither of these work in one test of mine. The output looks like

```
322 1000 alfa   pts/0
```

Where it goes wrong is the ` .*` at the end - there is no space following that `pts/0`.

The easy fix appears to be to just not require that space - `(\S+).*`. The `\S+` should be greedy, so the `.*` can only ever start with something that isn't matched by the `\S`.

Tho it is early in the morning here, so maybe I'm just not awake enough to deal with regexen.